apps/asap-server/test/controllers/research-outputs.test.ts

```diff
@@ -382,6 +383,41 @@ describe('ResearchOutputs controller', () => {
       expect(result.authors).toEqual(expectedAuthorsResponse);
     });
 
+    test('Should return a list of PM emails', async () => {
+      const researchOutputResponse = getSquidexResearchOutputGraphqlResponse();
+      nock(config.baseUrl)
+        .post(`/api/content/${config.appName}/graphql`, {
+          query: buildGraphQLQueryResearchOutput(researchOutputId),
+        })
+        .reply(200, { data: researchOutputResponse });
+
+      const result = await researchOutputs.fetchById(researchOutputId);
+      expect(result.pmsEmails).toEqual([
+        'pm1@example.com',
+        'pm2@example.com',
+        'multiple-pms-on-same-team@example.com',
+      ]);
+    });
+
+    test('PM emails should be deduplicated', async () => {
+      const researchOutputResponse = getSquidexResearchOutputGraphqlResponse();
+      nock(config.baseUrl)
+        .post(`/api/content/${config.appName}/graphql`, {
+          query: buildGraphQLQueryResearchOutput(researchOutputId),
+        })
+        .reply(200, { data: researchOutputResponse });
+
+      const result = await researchOutputs.fetchById(researchOutputId);
+
+      // Both these PMs are duplicated in the fixture
```

Comment:
instead of stating that in the comment i would leave the previous fixture as is and have a different setup specifically in this test